src/Csrf/CsrfViewInjection.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Yiisoft\Yii\View\Csrf;
+
+use Yiisoft\Router\UrlMatcherInterface;
+use Yiisoft\Yii\View\ContentParamsInjectionInterface;
+use Yiisoft\Yii\View\LayoutParamsInjectionInterface;
+use Yiisoft\Yii\View\MetaTagsInjectionInterface;
+
+class CsrfViewInjection implements
+    ContentParamsInjectionInterface,
+    LayoutParamsInjectionInterface,
+    MetaTagsInjectionInterface,
+    CsrfViewInjectionInterface
+{
+    public const DEFAULT_REQUEST_ATTRIBUTE = 'csrf_token';
+    public const DEFAULT_META_ATTRIBUTE = 'csrf';
+    public const DEFAULT_PARAMETER = 'csrf';
+
+    private UrlMatcherInterface $urlMatcher;
+
+    private string $requestAttribute = self::DEFAULT_REQUEST_ATTRIBUTE;
+    private string $metaAttribute = self::DEFAULT_META_ATTRIBUTE;
+    private string $parameter = self::DEFAULT_PARAMETER;
+
+    private ?string $csrfToken = null;
+
+    public function __construct(UrlMatcherInterface $urlMatcher)
+    {
+        $this->urlMatcher = $urlMatcher;
+    }
+
+    public function withRequestAttribute(?string $requestAttribute = null): CsrfViewInjectionInterface
+    {
+        $clone = clone $this;
+        $clone->requestAttribute = $requestAttribute ?? self::DEFAULT_REQUEST_ATTRIBUTE;
```

Comment:
```suggestion
    public function withRequestAttribute(string $requestAttribute): CsrfViewInjectionInterface
    {
        $clone = clone $this;
        $clone->requestAttribute = $requestAttribute;
```

Reply:
This is not convenient to use in this case: https://github.com/yiisoft/yii-demo/blob/master/src/Contact/ContactController.php#L62

Reply:
maybe it's better?
```suggestion
    public function withRequestAttribute(string $requestAttribute = self::DEFAULT_REQUEST_ATTRIBUTE): CsrfViewInjectionInterface
    {
        $clone = clone $this;
        $clone->requestAttribute = $requestAttribute;
```

Reply:
This is method from interface `CsrfViewInjectionInterface`, where not `self::DEFAULT_REQUEST_ATTRIBUTE`.